mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -546,6 +550,7 @@ private void doRenameMailbox(Mailbox mailbox, MailboxPath newMailboxPath, Mailbo
                 MailboxPath fromPath = new MailboxPath(from, subOriginalName);
                 sub.setName(subNewName);
                 mapper.rename(sub);
+                resultBuilder.add(new MailboxRenamedResult(sub.getMailboxId(), fromPath, sub.generateAssociatedPath()));
                 eventBus.dispatch(EventFactory.mailboxRenamed()
```

Comment:
Wow, this is a huge problem (not part of this PR): we are in a transaction that can rollback but we notify the listeners for every change while processing

Reply:
Can you open a separate issue about this @mbaechler ? (Both Apache JIRA and internal Gitlab?)